CONTRIBUTING.md

```diff
@@ -1,601 +1,501 @@
+## Development Setup
 
-Optionally, install language packs for Eclipse:
- * `Menu` --> `Help` --> `Install New Software`
- * Use the following update site:
-   ```
-   https://download.eclipse.org/technology/babel/update-site/latest/
-   ```
- * Select the language packs you want to install
- * By default, Eclipse uses the host operating system language (locale).
-   To force the use of another language, use the **-nl** parameter:
-   ```
-   eclipse -nl de
-   ```
+### Eclipse IDE Setup
 
+Eclipse IDE is **required** for Portfolio Performance development as the application is built on Eclipse RCP (Rich Client Platform).
 
-### Install Eclipse plug-ins
+**Prerequisites:**
+- **Java 21** - Required minimum version ([Download from Azul](https://www.azul.com/downloads/))
+- **[Eclipse IDE for RCP and RAP Developers](https://www.eclipse.org/downloads/packages/)** - Download the RCP package
 
-Optionally, install via the Eclipse Marketplace (drag and drop the *Install* button to your Eclipse workspace)
+**Essential Eclipse Plugins:**
 
-* [Eclipse PDE (Plug-in Development Environment)](https://marketplace.eclipse.org/content/eclipse-pde-plug-development-environment) (skip if you installed the *Eclipse IDE for RCP and RAP Developers*)
-* [Infinitest](https://marketplace.eclipse.org/content/infinitest)
-* [ResourceBundle Editor](https://marketplace.eclipse.org/content/resourcebundle-editor)
-* [Checkstyle Plug-In](https://marketplace.eclipse.org/content/checkstyle-plug)
-* [SonarLint](https://marketplace.eclipse.org/content/sonarlint)
-* [Launch Configuration DSL](https://marketplace.eclipse.org/content/launch-configuration-dsl)
-* [Darkest Dark Theme with DevStyle](https://marketplace.eclipse.org/content/darkest-dark-theme-devstyle) (Programming and chilling in DarkMode :sunglasses:)
+Install via Eclipse Marketplace (drag and drop the *Install* button to your workspace):
+- [Infinitest](https://marketplace.eclipse.org/content/infinitest) - Continuous testing
+- [ResourceBundle Editor](https://marketplace.eclipse.org/content/resourcebundle-editor) - Translation file editing
+- [Checkstyle Plug-In](https://marketplace.eclipse.org/content/checkstyle-plug) - Code style checking
+- [SonarQube for IDE](https://marketplace.eclipse.org/content/sonarqube-ide) - Code quality analysis
+- [Launch Configuration DSL](https://marketplace.eclipse.org/content/launch-configuration-dsl) - Launch configuration management
 
+**Install from Eclipse Simultaneous Release:**
 
-#### Install from the simultaneous release update site
+`Menu` → `Help` → `Install New Software` → Select `Latest Eclipse Simultaneous Release`:
+- **M2E PDE Integration** (under General Purpose Tools)
+- **Eclipse e4 Tools Developer Resources** (under General Purpose Tools)
```

Comment:
I am not sure if the is still correct.
Why is it only to needed to install e4 tools here?
And what about the other plugins.

I would say we leave it up to the contributor how to install eclipse. there are hundreds of sites out there.

Reply:
Thinking about it: I think we want to say some plugins are installed from the marketplace, some are installed from the Release update site.

I will update the notes